src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -36,8 +36,8 @@
 # trademark licensing policies.
 
 """
-PBS Professional hook for managing cgroups on Linux execution hosts.
-This hook contains the handlers required for PBS Professional to support
+OpenPBS hook for managing cgroups on Linux execution hosts.
+This hook contains the handlers required for OpenPBS to support
```

Comment:
Suggest you use PBS rather than OpenPBS here. The hook is not exclusive to OpenPBS.